src/coreclr/inc/corinfo.h

```diff
@@ -819,7 +819,6 @@ enum CORINFO_EH_CLAUSE_FLAGS
     CORINFO_EH_CLAUSE_FILTER    = 0x0001, // If this bit is on, then this EH entry is for a filter
     CORINFO_EH_CLAUSE_FINALLY   = 0x0002, // This clause is a finally clause
     CORINFO_EH_CLAUSE_FAULT     = 0x0004, // This clause is a fault clause
```

Comment:
```suggestion
    CORINFO_EH_CLAUSE_FAULT     = 0x0004, // This clause is a fault clause
    // UNUSED                                  = 0x0008,
```
Style nit: JIT/EE interface tends to keep unused bits explicit